pkg/traceql/engine_metrics.go

```diff
@@ -1097,8 +1110,9 @@ func (e *MetricsEvalulator) sampleExemplar(id []byte) bool {
 // MetricsFrontendEvaluator pipes the sharded job results back into the engine for the rest
 // of the pipeline.  i.e. This evaluator is for the query-frontend.
 type MetricsFrontendEvaluator struct {
-	mtx             sync.Mutex
-	metricsPipeline metricsFirstStageElement
+	mtx                sync.Mutex
+	metricsPipeline    metricsFirstStageElement
+	metricsSecondStage metricsSecondStageElement
```

Comment:
nit: these names stutter, we could do `firstStage`, `secondStage`

Reply:
renamed and trimmed metrics prefix